### DISCUSSION TITLE
Reverse selected publications order in about page

### DISCUSSION THREAD

**Discussion by alpinho** [alshedivat/al-folio#1950] [Q&A]
I would like to know how I can present my "Selected Publications" list on the About page in reversed order. I tried some solutions proposed in [this issue](https://github.com/alshedivat/al-folio/issues/47), but nothing worked. Please, note that I can reverse the order in the Publications list tab by changing the order of the years in `publications.md` file.

**Comment by george-gca** [comment]
I believe you are using an older version of our template, since we removed the list of years in the publications page, as you can see [here](https://raw.githubusercontent.com/alshedivat/al-folio/master/_pages/publications.md). But you can probably achieve this by setting some parameters for the query done in [selected_papers.html](https://github.com/alshedivat/al-folio/blob/master/_includes/selected_papers.html) for the [jekyll-scholar plugin](https://github.com/inukshuk/jekyll-scholar). More specifically, try this:

```html
{% bibliography -f {{ site.scholar.bibliography }} --group_by none --sort_by year,month --order descending --query @*[selected=true]* %}
```

**Reply by alpinho** [reply]
Thank you @george-gca for your feedback! However, I could not solve my problem. The suggested fix is messing a bit with my layout. This is what I have as default in `~/_includes/selected_papers.html`:

```         <div class="publications">
            <h2>Selected Publications</h2>
            {% bibliography -f papers -q @*[selected=true]* %}
          </div>
``` 

And this is what I have in `~/_pages/publications.md`:

```   
---
layout: page
permalink: /publications/
title: Publications
description: Publications in reversed chronological order.
years: [2023, 2021, 2020, 2019, 2018, 2015, 2014]
nav: true
nav_order: 1
---
<!-- _pages/publications.md -->
<div class="publications">

{%- for y in page.years %}
  <h2 class="year">{{y}}</h2>
  {% bibliography -f papers -q @*[year={{y}}]* %}
{% endfor %}

</div>

```   

I tried to apply the suggested code, namely replace the code in `~/_pages/publications.md` w/ the code above and change line in `~/_includes/selected_papers.html` also w/ the one written above. Besides, I tried to do some minor adjustments but nothing worked out well. Any other suggestion? Thx once more.

**Reply by george-gca** [reply]
Just tested the solution I proposed in the latest version of our code and is not working as expected. Just commented about this on [an issue in their repo](https://github.com/inukshuk/jekyll-scholar/issues/345).
